### ISSUE TITLE
UNKNOWN_OFFSET is changed from MACRO to const integer

### ISSUE THREAD

**Issue by gadget114514** [mchalupa/dg#170]
in order to use namespace, i guess it looks better that UNKNOWN_OFFSET macro in Offset.h is change to const integer like const uint64_t UNKNOWN_OFFSET = ~((uint64_t)0) or it would be a public static class member of struct Offset. the instant change would cause many compilation error.

**Comment by mchalupa** [comment]
This sounds reasonable, thanks. I'll postpone making this change once the #163 is merged so that it won't break it too much.

**Comment by mchalupa** [comment]
This change has been incorporated.
